app/assets/javascripts/discourse/app/routes/signup.js

```diff
@@ -1,14 +1,19 @@
 import { action } from "@ember/object";
 import { next } from "@ember/runloop";
 import { service } from "@ember/service";
+import { defaultHomepage } from "discourse/lib/utilities";
 import DiscourseRoute from "discourse/routes/discourse";
 
 export default class SignupRoute extends DiscourseRoute {
   @service router;
   @service siteSettings;
 
   beforeModel() {
-    this.showCreateAccount();
+    if (!this.siteSettings.experimental_full_page_login) {
+      this.showCreateAccount();
+    } else if (this.siteSettings.invite_only) {
+      this.router.replaceWith(`/${defaultHomepage()}`).followRedirects();
```

Comment:
What is this change supposed to do? When the site has `invite_only` and `experimental_full_page_login` enabled, it is supposed to redirect the user from `/signup` to a different URL? That doesn't seem to be working for me locally using this branch. 

Not a blocker though.

Reply:
Are you sure you have the latest version? I found out about this when running the system tests. The change is supposed to redirect you to the homepage if signups are not enabled. Previously, it would show the form. I just refactored it to cover all cases, tho. Maybe it's working now?

Reply:
You're right, I hadn't pulled the latest commits. Looks good now.